backend/plugin/advisor/mysql/advisor_online_migration.go

```diff
@@ -81,26 +71,52 @@ func (*OnlineMigrationAdvisor) Check(ctx advisor.Context, _ string) ([]*storepb.
 			}
 			if tableRows >= minRows {
 				adviceList = append(adviceList, &storepb.Advice{
-					Status:  checker.level,
-					Code:    advisor.AdviseOnlineMigration.Int32(),
-					Title:   checker.title,
-					Content: fmt.Sprintf("Estimated table row count of %q is %d exceeding the set value %d. Consider using online migration for this statement", fmt.Sprintf("%s.%s", resource.Schema, resource.Table), tableRows, minRows),
-					StartPosition: &storepb.Position{
-						Line: int32(stmt.BaseLine),
-					},
+					Status:        checker.level,
+					Code:          advisor.AdviseOnlineMigrationForStatement.Int32(),
+					Title:         checker.title,
+					Content:       fmt.Sprintf("Estimated table row count of %q is %d exceeding the set value %d. Consider using online migration for this statement", fmt.Sprintf("%s.%s", resource.Schema, resource.Table), tableRows, minRows),
+					StartPosition: checker.start,
+					EndPosition:   checker.end,
 				})
 			}
 		}
 	}
 
+	if len(adviceList) > 1 {
```

Comment:
Add more comments/explanations to the following code especially for the conditions?